js/modules/inputValidation.js

```diff
@@ -0,0 +1,26 @@
+const elemsToValidate = document.body.querySelectorAll(".isValid");
+const ERROR_DELAY = 3000;
+
+elemsToValidate.forEach(elem => {
+	
+	const input = elem.querySelector(".validate");
+
+	input.addEventListener("invalid", function(event){
+		event.preventDefault();
+		if(elem.dataset.isErrorShown === "true") {
+			return;
+		}
+		showError(elem);
+		setTimeout(hideError, ERROR_DELAY, elem);
+	})
+})
+
+function showError(invalidItem){
+	invalidItem.classList.add("error");
+	invalidItem.dataset.isErrorShown = true;
```

Comment:
why do you need this?

Reply:
Defense from durak... Disable repeated showing error,  while error is shown.  It's needed.

Reply:
![image](https://user-images.githubusercontent.com/37267977/216141157-0f9cd7bb-c185-4b78-b2b6-debb71d8d739.png)

Reply:
Done